crates/router/src/core/payments/helpers.rs

```diff
@@ -1197,7 +1197,11 @@ pub async fn make_ephemeral_key(
 ) -> errors::RouterResponse<ephemeral_key::EphemeralKey> {
     let store = &state.store;
     let id = utils::generate_id(consts::ID_LENGTH, "eki");
-    let secret = format!("epk_{}", &Uuid::new_v4().simple().to_string());
+    let secret = format!(
+        "{}_{}",
+        consts::EPHEMERAL_KEY_PREFIX,
+        &Uuid::new_v4().simple().to_string()
```

Comment:
@dracarys18 Since this is a secret, should we use [`generate_cryptographically_secure_random_string()`](https://github.com/juspay/hyperswitch/blob/61bacd8c9590a78a6d5067e378bfed6301d64d07/crates/common_utils/src/crypto.rs#L357-L364) to generate the secret instead?

Reply:
@SanchithHegde we can use this 

cc: @hrithikesh026

Reply:
Should I pass 32 as `length` for `generate_cryptographically_secure_random_string(length: usize)` ?

Reply:
Yeah @hrithikesh026